apps/launch/hooks/useGetMyStakingContracts.ts

```diff
@@ -170,12 +172,12 @@ export const useGetMyStakingContracts = () => {
 
         return {
           id: instanceAddresses[index],
+          chainId: networkId,
           name: metadata.name,
           description: metadata.description,
           template: CONTRACT_TEMPLATES[0].title,
           isNominated:
-            nominees.some((nominee) => instanceAddressesInBytes32.includes(nominee.account)) ??
-            false,
+            nominees.findIndex((item) => item.account === instanceAddressesInBytes32[index]) !== -1,
```

Comment:
is that the previous one always returned true even if the current contract is not in the nominees list. because we just simply check if at least one nominee is in the contracts addresses list (which is always true if at least one contract is nominated), but should check if current address is in nominees.

+ easier to read.